packages/api/src/command/medical/patient/__tests__/store-query-cmd.ts

```diff
@@ -39,14 +44,23 @@ export const documentQueryProgress: DocumentQueryProgress = {
   convert: makeProgress(),
 };
 
-export const consolidatedQuery: QueryProgress = {
-  status: "processing",
-  startedAt: new Date(),
-};
+export function makeConsolidatedQueryProgress(
+  params?: Partial<ConsolidatedQuery>
+): ConsolidatedQuery {
+  return {
+    requestId: params?.requestId ?? requestId,
+    status: params?.status ?? "processing",
+    startedAt: params?.startedAt ?? new Date(),
+    resources: params?.resources ?? [],
+    conversionType: params?.conversionType ?? "json",
+    dateFrom: params?.dateFrom ?? dayjs().subtract(10, "years").format(ISO_DATE),
+    dateTo: params?.dateTo ?? dayjs().add(1, "day").format(ISO_DATE),
```

Comment:
Just FYI: FakerJS has functions for this:
- [past](https://fakerjs.dev/api/date.html#past)
- [recent](https://fakerjs.dev/api/date.html#recent)

Tip: first define `to` and use it as a reference to `past` when building `from`, so its always before `to`/`recent`

Reply:
Cool didnt know this